utils/githooks/pre-commit.d/20-isort.sh

```diff
@@ -4,6 +4,13 @@
 
 set -ue
 
+# Disable isort check because it doesn't work
+# - it needs to run with -rc
+# - it conflicts with pylint checks
+# - ~100 files are flagged with import errors
+# - it's really slow
```

Comment:
How slow is it for you? it is pretty fast for me:

(ai-1775)yinsun@s07ln0001:~/src/daos$ time isort --jobs 8 --check-only .
Skipped 5 files

real    0m0.604s
user    0m0.956s
sys     0m0.282s

Reply:
`real    0m3.815s`
But it seems it's because so many files are flagged with warnings for me